### PR TITLE
add workflow to auto generate template patch

### PR DESCRIPTION
**Pull request by dkonieczek**

### CODE REVIEW

**Review by korgon — CHANGES_REQUESTED** [review]
I like your idea of querying the org for applicable template repos - very nice.

File: .github/workflows/patchTemplate.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      - name: Setup Node
+        id: setup-node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install Snapfu
+        run: |
+          npm install -g snapfu
+
```
Comment (korgon) [inline .github/workflows/patchTemplate.yml:62]: Can this be done outside of the matrix loop to prevent it from happening over and over unnecessarily?

Comment (dkonieczek) [reply .github/workflows/patchTemplate.yml:62]: Looks like both the 'needs' dependancy and each entry of the matrix runs in a separate environment so I'm not seeing a way.

Comment (korgon) [reply .github/workflows/patchTemplate.yml:62]: Alright - just thought it would be good to check :)

**Review by korgon — APPROVED** [review]

**Pull request merged**